src/components/content/content.ts

```diff
@@ -782,6 +785,11 @@ export class Content extends Ion implements OnDestroy, AfterViewInit, IContent {
       this._cBottom += this._tabbarHeight;
     }
 
+    // Refresher uses a border which should be hidden unless pulled
+    if (this._hasRefresher === true) {
```

Comment:
let's trust TS a little bit :D it should be a valid boolean. I think it is ok to drop the "=== true"